dotcom-rendering/src/web/components/SubheadingBlockComponent.tsx

```diff
@@ -1,19 +1,36 @@
-import { unwrapHtml } from '../../model/unwrapHtml';
-import { RewrappedComponent } from './RewrappedComponent';
+import { jsx } from '@emotion/react';
+import { JSDOM } from 'jsdom';
+import type { ReactNode } from 'react';
+import { Fragment } from 'react';
 
 type Props = { html: string };
 
+const parseHtml = (html: string): DocumentFragment => JSDOM.fragment(html);
+
+function isElement(node: Node): node is Element {
+	return node.nodeType === 1;
+}
+
+const getAttrs = (node: Node): NamedNodeMap | undefined =>
+	isElement(node) ? node.attributes : undefined;
+
+const buildElementTree = (node: Node): ReactNode => {
+	if (node.nodeType === node.ELEMENT_NODE) {
+		const element = node as Element;
+		return jsx(element.tagName.toLowerCase(), {
```

Comment:
I think you have the `isElement` function above to provide the `node is Element` type predicate[^1], so this could be written without the type cast as:

```suggestion
	if (isElement(node)) {
		return jsx(node.tagName.toLowerCase(), {
```

However, it might be preferable to sidestep both the type cast and the type predicate by using `instanceof` (as noted by @mxdvl):

```suggestion
	if (node instanceof Element) {
		return jsx(node.tagName.toLowerCase(), {
```

which TS can use to automatically narrow the type for you[^2]. This is available because `Element` is a sub-type of `Node`[^3].

[^1]: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates
[^2]: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#instanceof-narrowing
[^3]: https://developer.mozilla.org/en-US/docs/Web/API/Element